cluster/cluster_impl/broadcast_cluster.go

```diff
@@ -39,6 +39,7 @@ func NewBroadcastCluster() cluster.Cluster {
 	return &broadcastCluster{}
 }
 
+// Join would return baseClusterInvoker instance
```

Comment:
the comment should be `Join returns a baseClusterInvoker instance`.

And the following comment should change as the same.